### DISCUSSION TITLE
Anyone got a working 5 Plus with Coral Dual Edge TPU ?

### DISCUSSION THREAD

**Discussion by ymurawski** [Joshua-Riek/ubuntu-rockchip#1150] [Q&A]
Hi, is there anyone who has a a running Dual Edge TPu ising the PCIe Slot on the top of the board ?
When i enter "lspci | grep Edge" or "lspci -nn | grep 089a" there is just 1 TPU showing up.

I installed the driver with this link https://coral.ai/docs/m2/get-started/

Kernel version
6.1.0-1025-rockchip

SBC model
Orangepi 5 Plus

**Comment by Deiiji** [comment]
I am waiting for mine but they tell me minimum a month to wait... I think, it will need an overlay but as long I can't try, I can't tell.

**Comment by Pippowicz** [comment]
I know that´s an older post but I am pretty sure that´s because the Coral Mini PCIe Accelerators need 2 PCIe Lanes and Wifi Modules typically need only one PCIe Lane for Wifi and USB for BT - so there is only one PCIe Link on the Upper Header causing only one TPU to show up.
